packages/search/src/test/use-fuzzy-search.tsx

```diff
@@ -0,0 +1,63 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
```

Comment:
Do we need the `React` import for anything? I thought we got rid of those at some point.